scim-core/src/main/java/org/apache/directory/scim/core/repository/PatchHandlerImpl.java

```diff
@@ -171,6 +171,18 @@ private static void checkMutability(Attribute attribute, Object currentValue) th
     }
   }
 
+  private static void checkPrimary(String subAttributeName, Collection<Map<String, Object>> items, Object value)
+  {
+    if (subAttributeName.equals("primary") && value.equals(true)) {
```

Comment:
@bdemers where would be a good place to declare `static final PRIMARY = "primary"`? 
`Schema` maybe?